src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -132,3 +132,26 @@ function(ov_add_onednn)
 endfunction()
 
 ov_add_onednn()
+
+if(ENABLE_LLMDNN)
+    # llmdnn needs intel compiler to be compiled
+    find_program(ICX_RESULT icx)
+    if(NOT ICX_RESULT)
+        message(FATAL_ERROR "intel compiler 2023.0+ is needed!")
```

Comment:
what Intel compiler specific do we have in this library?

Reply:
2023.0 and 2023.1 are tested, the version check is done [here](https://github.com/openvinotoolkit/openvino/blob/6655e61fc9df4481da55ebc57a786cdbbf184e50/src/plugins/intel_cpu/thirdparty/llmdnn/CMakeLists.txt#L30).

Reply:
I mean we don't want to build with Intel compiler, we should rely on MSVC on Windows.

Reply:
Removed icc in windows.